geode-redis/src/test/java/org/apache/geode/redis/internal/executor/string/StringSetExecutorJUnitTest.java

```diff
@@ -213,4 +229,34 @@ public void testSET_XXandNX_inSameCommand_ReturnsError() {
     assertThat(response.toString()).contains(RedisConstants.ERROR_SYNTAX);
   }
 
+  @Test
+  public void testSET_unknownOption_ReturnsError() {
```

Comment:
I'm not sure these tests should be added here. If you added them instead to SetIntegrationTest then I'd feel better because they would also run against native redis.

Reply:
done.  (1 test deleted instead of  moved as the same basic test already existed in SetIntegrationTest )

Reply:
done